Sources/Sentry/SentryPerformanceTrackingIntegration.m

```diff
@@ -0,0 +1,23 @@
+#import "SentryPerformanceTrackingIntegration.h"
+#import "SentryUIPerformanceTracker.h"
+
+@interface
+SentryPerformanceTrackingIntegration ()
+
+@property (nonatomic, weak) SentryOptions *options;
```

Comment:
To be honest I just copied from SentryAutoBreadcrumbTrackingIntegration.m.

I don't think it makes any difference since options in SentryHub is never disposed. But maybe we should change this to avoid future problems.